index.html

```diff
@@ -138,6 +138,26 @@ <h2>CapturedMouseEvent interface</h2>
         more scrutiny in which information may safely be exposed. At present, only information
         already available to the capturing application is exposed - the mouse coordinates.
       </p>
+      <p>
+        If a <a data-cite="screen-capture/#dfn-capture-session">capture-session</a>
+        has an associated {{CaptureController}} whose
+        <a data-cite="screen-capture/#dfn-source">`[[Source]]`</a> is not
```

Comment:
Please use `{{CaptureController/[[Source]]}}`.

Reply:
This is reporting 

> Couldn't find "[[Source]]", for "CaptureController", in this document or other cited documents: [dom], [html], [infra], [mediacapture-streams], [permissions], [permissions-policy], [screen-capture], [uievents], and [webidl].

I'm not sure why, maybe because it's an internal slot not a regular attribute.

Reply:
Let's make it a link, then, and try to export it from the original spec as a low-priority follow-up.

Reply:
OK so I understand this CL is ready to land then? Can you please merge it?

Reply:
Could you rebase it on top of tip-of-tree, run the formatter and ping me to merge?

Reply:
Done.